marathon_lb.py

```diff
@@ -1330,7 +1330,8 @@ def do_reset(self):
                     logger.info('stopping event processor thread')
                     return
                 if not self.__pending_reset:
-                    self.__condition.wait()
+                    if not self.__condition.wait(60):
```

Comment:
I thought about making this configurable, but there are already a million config params so I'd rather not.